### ISSUE TITLE
Internal: I'd like to tell you something else., row 107

### ISSUE THREAD

**Issue by plocket** [SuffolkLITLab/docassemble-MAVirtualCourt#491]
# Survey result on row 107

**Report type:** I'd like to tell you something else.

**Contact info**: Caroline

---

### Bug

Question | Answer
---|---
**Question** | 
**Actions** | 
**Expected** | 
**Actual** | 
**Feedback** | 

---

### Confusion feedback

Question | Answer
---|---
**Question** | 
**Confusion** | 
**Feedback** | 

---

### Something else

Uncategorized Feedback:
Page id: fled from addresses, v0.96
Not for MVP but when asking about previous addresses plaintiff fled because of abuse, although we ask so that plaintiff can file in another court from their current local court people do not not know why we are asking - maybe just add to subquestion - because many people fled abuse in another state or another country -  @kateleebarry ??

---

### Test report

Question | Answer
---|---
**Link to spreadsheet** | 
**Test id** | 
**Tester** | 
**Question** | 
**Input** | 
**Expected** | 
**Actual** | 
**Link** | 
**Pic** | 
**Notes** | 

**Error message**
```

```

---

### Thank you!

**Comment by kateleebarry** [comment]
Agreed an explanation would be nice.  Agreed this would be good for version 2.

**Comment by nonprofittechy** [comment]
The international address input seems really important. I'll try to get this done soon.
